randovania/generator/hint_distributor.py

```diff
@@ -139,6 +140,16 @@ def choose_feature(
         return feature
 
 
+class HintSuitability(enum.Enum):
+    """
+    How interesting a pickup is for hinting.
+    """
+
+    INTERESTING = enum.auto()
+    LESS_INTERESTING = enum.auto()
+    NOT_INTERESTING = enum.auto()
```

Comment:
I'd make this an `IntEnum` with explicit values, so that `INTERESTING > LESS_INTERESTING > NOT_INTERESTING`